src/Eurofurence.App.Domain.Model/ArtistsAlley/UserStatus/ArtistAlleyUserPenaltyRecord.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Eurofurence.App.Domain.Model.ArtistsAlley
+{
+
+    
+    /// <summary>
+    /// Record for storing
+    /// </summary>
+    [Table("ArtistAlleyUserPenalties")]
+    public class ArtistAlleyUserPenaltyRecord : EntityBase
+    {
+
+        public ArtistAlleyUserPenaltyRecord()
+        {
+            Penalty = UserPenalties.OK;
+        }
+
+        /// <summary>
+        /// Possible penalties of a user
+        /// </summary>
+        public enum UserPenalties
+        {
+            BANNED = 0,
```

Comment:
I'd suggest having the "good standing" (i.e. "OK") status be 0 as the default, so any non-zero status can easily be identified as having some kind of penalty (e.g. "BANNED" or maybe other's in the future).